ParsingModules/ToCHelper/TwoColumnParser.py

```diff
@@ -0,0 +1,226 @@
+from enum import unique, Enum, auto
+from typing import List, Union
+
+from Certificate import Certificate
+
+
+@unique
+class TWO_COL_PARSER_STATE(Enum):
+    EMPTY = auto(),
+    CHAPTER = auto(),
+    NAME = auto(),
+    SPACE_PLACEHOLDER = auto(),
+    PAGE_NUMBER = auto(),
+    END = auto(),
+    ERROR = auto(),
+    LEFT = auto(),
+    RIGHT = auto(),
+    NEXT_LINE = auto(),
+    SKIP_LINE = auto()
+
+
+class TwoColumnParser:
+    _CONFIG_SPACE_BAR = 16
+    _CONFIG_DOTS = 3
+
+    _CONFIG_ALLOWED_CHARS = ["(", "[", "]", ")"]
+
+    def __init__(self, content: str, certificate: Certificate):
+        self._content = content
+        self._STATE_LEFT = TWO_COL_PARSER_STATE.EMPTY
+        self._STATE_RIGHT = TWO_COL_PARSER_STATE.EMPTY
+        self._STATE = TWO_COL_PARSER_STATE.EMPTY
+        self._SIDE = TWO_COL_PARSER_STATE.LEFT
+        self._left_chapter = None
+        self._left_chapter_name = ''
+        self._left_page_number = None
+        self._right_chapter = None
+        self._right_chapter_name = ''
+        self._right_page_number = None
```

Comment:
All of these could be initialized to an empty string instead of None. Later in the code, you use the `+=` operator, which would throw an exception if the variable was still None.